### ISSUE TITLE
should handle wrong passwords

### ISSUE THREAD

**Issue by nagualcode** [guh/nymea-networkmanager#13]
Project is very nice.
But if enter wrong pass, there is no feedback, ou return option.

**Comment by jaumard** [comment]
Agreed that it need a dedicated error code for wrong password. For now `unknown` errors must be considered as wrong password.
